creator-node/src/routes/nodeSync.js

```diff
@@ -24,56 +24,92 @@ module.exports = function (app) {
    * }
    */
   app.get('/export', handleResponse(async (req, res) => {
+    // TODO - allow for offsets in the /export
+    const limit = 25000
     const walletPublicKeys = req.query.wallet_public_key // array
     const dbOnlySync = (req.query.db_only_sync === true || req.query.db_only_sync === 'true')
 
     const transaction = await models.sequelize.transaction()
     try {
       // Fetch cnodeUser for each walletPublicKey.
-      const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction })
+      const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction, raw: true })
       const cnodeUserUUIDs = cnodeUsers.map((cnodeUser) => cnodeUser.cnodeUserUUID)
+      const cnodeUserUUID = cnodeUserUUIDs[0] // assume we only have a single wallet being exported for now
```

Comment:
small thing but since this breaks sync spec (even though we never call it with more than one wallet), can we also have export return error if multiple wallets are requested?

Reply:
yep i can definitely do this

Reply:
reverted